app/views/layouts/application.html.erb

```diff
@@ -9,8 +9,14 @@
 
     <%= stylesheet_link_tag "https://www-media.stanford.edu/su-identity/css/su-identity.css" %>
     <%= stylesheet_link_tag "https://fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,700" %>
+    <%= stylesheet_link_tag "https://cdn.datatables.net/1.10.23/css/dataTables.bootstrap5.min.css" %>
     <%= stylesheet_pack_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
     <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
+    <script src="https://code.jquery.com/jquery-3.5.1.js"></script>
```

Comment:
Please don't add jquery. It's a very big library and we don't need to add all that weight just to sort some tables.  It's not very challenging to just roll your own here.

Reply:
take a look at https://medium.com/javascript-in-plain-english/easy-table-sorting-with-javascript-370d8d97cad8

Reply:
Thanks for the tip, that was going to be my next question based on something I read today.